go.mod

```diff
@@ -152,6 +151,9 @@ require (
 )
 
 replace (
+	github.com/circlefin/noble-cctp => github.com/noble-assets/cctp v0.0.0-20231017104045-821e63c6c4cb
+	github.com/circlefin/noble-fiattokenfactory => github.com/noble-assets/fiattokenfactory v0.0.0-20231017103812-a2afc92603fc
```

Comment:
I assume we will remove these replaces and the associated changes like the GITAUTH -> SSH_NOBLE before merging?